frontend/src/tests/UsernameInput.spec.ts

```diff
@@ -184,6 +294,38 @@ describe('UserNameInput', () => {
         expect(wrapper.find('#lastName').element.textContent).toContain(
             LASTNAME
         );
+        expect(wrapper.find('#organizationName').element.textContent).toContain(
+            BUSINESSLEGALNAME
+        );
+    });
+
+    it.skip('Should show the user identity card with permission when BCEID user is not allowed to search', async () => {
```

Comment:
once we figured how to mock the axios error could remove the "skip" and run this test